modules/@apostrophecms/express/index.js

```diff
@@ -158,9 +145,10 @@ const bodyParser = require('body-parser');
 const expressSession = require('express-session');
 const cookieParser = require('cookie-parser');
 const qs = require('qs');
+const expressBearerToken = require('express-bearer-token');
 
 module.exports = {
-  init(self, options) {
+  async init(self, options) {
```

Comment:
`async` is not necessary here I think (no `await`)

Reply:
True